src/ImageHotspots.js

```diff
@@ -144,17 +155,22 @@ class ImageHotspots extends React.Component {
       <div ref={this.container} style={containerStyle}>
         {
           src &&
-            <img src={src} alt={alt} onLoad={this.onImageLoad} style={imageStyle} />
+          <img src={src} alt={alt}
+            onLoad={this.onImageLoad}
+            onMouseDown={evt => this.startDrag(evt)}
+            onMouseMove={evt => {
+              if (this.state.isDragging) {
+                this.whileDrag(evt)
+              }
+            }}
+            onMouseUp={this.stopDrag}
+            style={imageStyle} />
         }
         {
           !hideHotspots && hotspots &&
-            <div style={hotspotsStyle}>
```

Comment:
@Filipe, I had to remove the div around the hotspot because, this is not allowing me to trigger the mouse event on the image, instead the mouse event is going to the hotspot. Bryan helped me in figuring this out.